boat_simulator/nodes/physics_engine/fluid_generation.py

```diff
@@ -0,0 +1,37 @@
+"""Generator for fluid vectors used in the physics engine."""
+
+import numpy as np
+from numpy.typing import NDArray
+
+from boat_simulator.common.generators import VectorGenerator
+from boat_simulator.common.types import Scalar
+
+
+class FluidGenerator:
+    def __init__(self, generator: VectorGenerator):
+        self.__generator = generator
+        self.__velocity = np.array([0, 0])
```

Comment:
Should this be zero vector, or should we use the first value generated by the generator?